### PR TITLE
ME-65: Add a "simple_sign_connection" table for simple sign members

### PR DESCRIPTION
**Pull request by niho**
I've created a separate PR for this step, as it is independent of the importer in #481.

### CODE REVIEW

**Review by bystam — APPROVED** [review]

**Comment by niho** [comment]
I'm just thinking if it's confusing to have it named "Connection"? It's not really a credential, but it's not really a connection either.

**Comment by bystam** [comment]
> I'm just thinking if it's confusing to have it named "Connection"? It's not really a credential, but it's not really a connection either.

Do you have a suggestion for a better name before we merge? `SimpleSignOrigin`?

File: member-service/src/test/java/com/hedvig/auth/AuthTest.kt [hunk 1]
```diff
@@ -441,4 +446,181 @@ internal class AuthTest {
...
+    fun `can delete SimpleSignConnection`() {
+        val connection = simpleSignConnectionRepository.saveAndFlush(
+            SimpleSignConnection(
+                user = User(associatedMemberId = "test-member-id"),
+                personalNumber = "1234",
+                country = "NO"
+            )
+        )
+        simpleSignConnectionRepository.delete(connection)
+        val connectionAfterDeletion = zignSecCredentialRepository.findByIdOrNull(connection.id)
```
Comment (niho) [inline member-service/src/test/java/com/hedvig/auth/AuthTest.kt:593]: I was splitting up the `AuthTest` file in #480 and rebased that branch on this branch. I then noticed this little nasty error. The `SimpleSignConnection` test is using the `zignSecCredentialRepository`!

Comment (bystam) [reply member-service/src/test/java/com/hedvig/auth/AuthTest.kt:593]: Oops! Will you fix it, or should I? Also did you have an opinion on the other name? SimpleSignOrigin that is.

**Pull request merged**